### ISSUE TITLE
Post pagination

### ISSUE THREAD

**Issue by adamf321** [moxie-leean/ng-cms#10]
We should have some functionality for pagination with the collection endpoint (and authros or any other which return multiple items). It should work if you need to add the posts to the bottom of the list, or if you need to scroll through pages one at a time.

Here's a start which might be useful:
https://gist.github.com/adamf321/34d7ccc75b83f3ad939141606444bb31

**Comment by mitogh** [comment]
Other things that this need to improve.
- Loading state
- Make it work on any API structure at the moment only works with the one defined in collection.

**Comment by adamf321** [comment]
Yes, loading animation is a good one - added here: https://github.com/moxie-leean/ng-patterns/issues/11

Is the other necessary to work with authors for example? So long as it works for all of our api's we're ok for now...
